src/snowcli/cli/common/cli_global_context.py

```diff
@@ -219,6 +227,10 @@ def verbose(self) -> bool:
     def experimental(self) -> bool:
         return self._manager.experimental
 
+    @property
+    def project_definition(self):
+        return self._manager.project_definition
```

Comment:
Is it ok to have `project_definition` property twice?

Reply:
It is in a different class :)